src/components/AppSettingsModal/AppSettingsModal.js

```diff
@@ -0,0 +1,72 @@
+/* eslint-disable react/prop-types */
```

Comment:
Don't need this comment here anymore; added import for `PropTypes`

```suggestion
import React, { useState } from 'react'
import PropTypes from 'prop-types'
```